lib/services.identity.js

```diff
@@ -376,6 +432,60 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       404: 'The identity was not found on the system.'
     }
   });
+
+  app.delete(idPath, ensureAuthenticated, function(req, res, next) {
```

Comment:
@dlongley @davidlehn Presently an identity that is not 'active' is not visible to the admin interface.  Do we need to have undelete functions in the admin UI?

We do need to disallow deletion of ones own identity in the UI.